### PR TITLE
Support for signed values.

### PR DESCRIPTION
**Pull request by OrangeTux**

### CODE REVIEW

**Comment by OrangeTux** [comment]
This PR already includes some code related to #11.

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    
+  from umodbus import conf
+
+  conf.SIGNED_VALUES = True 
+
+
+.. module:: umodbus.config
+
+.. autoclass:: Config
+    :members:  SIGNED_VALUES
```
Comment (jaapz) [inline docs/source/configuration.rst]: I think a config member is missing here?

Comment (OrangeTux) [inline docs/source/configuration.rst]: No, but I understand the confusion. I already included some code for a new feature: 32 bit values. The `BIT_SIZE` settings is related to this issue, but isn't functional yet. Currently bit size is 16 bit always, ignore this setting. That is the reason for not in including this setting in the docs right now.

**Comment by jaapz** [comment]
LGTM :+1:

**Pull request merged**